### PR TITLE
Add temp blog link and have a working router for homePage

### PR DESCRIPTION
**Pull request by oscar-leung**
https://user-images.githubusercontent.com/15118391/140434801-9eac031f-f42b-4453-a3a0-c3d0713f0142.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router": "^6.0.0",
```
Comment (josekim) [inline package.json:14]: You just need one, you can delete react-router

File: src/App.js [hunk 2]
```diff
@@ -21,6 +29,19 @@ function App() {
         </a>
         <Button variant="contained"> Hello world!</Button>
         <NavBar></NavBar>
+        {/* <HomePage></HomePage> */}
+        {/* <BlogPage /> */}
+
+        <Router>
+          <div>
```
Comment (josekim) [inline src/App.js:36]: remove this

File: src/App.js [hunk 3]
```diff
@@ -21,6 +29,19 @@ function App() {
         </a>
         <Button variant="contained"> Hello world!</Button>
         <NavBar></NavBar>
```
Comment (josekim) [inline src/App.js:31]: everything above should really be its own component,  Can you let me know what the purpose is of all the stuff above? or do you plan to delete it?

File: src/pages/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+
+const Root = styled("div")(({ theme }) => ({
```
Comment (josekim) [inline src/pages/HomePage.js:4]: why not just use` <Box sx={} > `

File: src/App.js [hunk 5]
```diff
@@ -21,6 +29,19 @@ function App() {
...
+            <Link to="/home">Homepage</Link>
+            <Routes>
+              <Route path="/home">
+                <HomePage />
+              </Route>
+            </Routes>
+          </div>
+        </Router>
       </header>
     </div>
```
Comment (josekim) [inline src/App.js:46]: i would add themeprovider for material ui inside of app component

File: src/App.js (hunk 3 above)
Comment (josekim) [reply src/App.js:31]: App component shouldn't have jsx in it, You should really be calling components at the top level components.

**Comment by josekim** [comment]
just merge what you have and address my feedback in your next PR please.  Thanks

File: src/App.js (hunk 3 above)
Comment (oscar-leung) [reply src/App.js:31]: I was planning to delete it

File: src/pages/HomePage.js (hunk 4 above)
Comment (oscar-leung) [reply src/pages/HomePage.js:4]: What do you mean?

Comment (josekim) [reply src/pages/HomePage.js:4]: https://mui.com/components/box/

I would use this component for some of the  css you are doing,  i would not use the styled theme that you were doing in the component.

**Pull request merged**